docs/developer/eventing/sources/sinkbinding/getting-started.md

```diff
@@ -186,11 +186,10 @@ Create a `SinkBinding` object that directs events from your subject to the sink.
     Create a `SinkBinding` object by running:
 
     ```bash
-    kn source binding create <name> \
-      --namespace <namespace> \
-      --subject "<subject>" \
-      --sink <sink> \
-      --ce-override "<cloudevent-overrides>"
+    kn source binding create bind-heartbeat \
+     --subject "Job:batch/v1:app=heartbeat-cron" \
+     --sink event-display \
+     --ce-override "sink=bound"
```

Comment:
I think this command was valid. This command has placeholders for users to input their own values. Could you replace this command please?

The [issue](https://github.com/knative/docs/issues/4178#issuecomment-912752920) is referring to the example command on lines 211-215. If this example command is causing confusion, it might make sense to just delete it.